prx_simulation/state_msg[] plant_configs
prx_simulation/state_msg[] obstacle_configs
string node_name
float64 timestamp